app/frontend/src/cljs/teet/ui/buttons.cljs

```diff
@@ -75,6 +75,11 @@
   (util/make-component Button {:variant        :contained
                                :disable-ripple true
                                :color          :primary}))
+(def large-button-primary
+  (util/make-component Button {:variant        :contained
```

Comment:
indentation

Reply:
I will not fix this, because the indentation has the same format in the whole file